packages/manager/modules/sign-up/src/form/form.controller.js

```diff
@@ -80,6 +80,14 @@ export default class SignUpFormCtrl {
       this.model,
     );
 
+    const legalformOrder = {
```

Comment:
Can't the order be handled by the API ?

Reply:
We can request this kind of enhancement.
Since this ticket already received a validation I will merge it in order to be able to deploy it.
Thank you!